### ISSUE TITLE
A11y_code.visualstudio.com_DownloadInsiders_AI4D: The interactive elements such as "Mac", "Linux", etc. have color contrast ratio less than 4.5:1.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-docs#2794]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://code.visualstudio.com](https://code.visualstudio.com) 
Windows Version: Windows10
Browser Version: 
Microsoft Edge 42.17134.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated: MAS 1.4.3

**Repro Steps:**
Navigate to the url: [https://code.visualstudio.com/insiders/](https://code.visualstudio.com/insiders/) 
follow below steps:
1.	Launch the application Url: [https://code.visualstudio.com](https://code.visualstudio.com)
2.	Navigate to "Download" button.
3.	Scroll down the page and hit "Insiders build" link.
4.	Navigate through the "Download Insiders" screen and navigate to "Mac", "Linux", etc. interactive elements.
5.	Check it's color contrast ratio.

**Actual:**
The interactive elements such as "Mac", "Linux", etc. have a color contrast ratio of 4.179:1 which is  not more than 4.5:1 as per the standard color contrast ratio which is incorrect.

**Expected:**
The color contrast ratio of the interactive elements should be more the 4.5:1 when compare with the background color and the colour of the text.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
If the color contrast of the interactive elements fails as compared to its background color then the low vision users will have problems accessing them.

**MAS Reference:**
MAS 1.4.3 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6})

**Attachment for Reference:**
![Code visualstudio com_DownloadInsiders_K4D Color contrast ratio fails](https://user-images.githubusercontent.com/52036772/60007660-5f4f4b80-9690-11e9-8bf9-9b38001c824a.png)

**Comment by octref** [comment]
The color is `#0072BE`, not `#337AB7`.

![image](https://user-images.githubusercontent.com/4033249/60831339-2cb45100-a16e-11e9-87da-3a0463e388cf.png)

![image](https://user-images.githubusercontent.com/4033249/60831384-4bb2e300-a16e-11e9-8e00-d9be160ddf0e.png)
